content/docs/guides/contribute/license-and-set-up.en.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "License and set-up"
+linkTitle: "License and set-up"
+weight: 2
+description: How to set up your development environment? What does our license involve?
+---
+
+## Our license: LGPLv3
+
+The source code of OSRD is available under [the LGPLv3 license](https://choosealicense.com/licenses/lgpl-3.0/).
+By contributing to the codebase, you consent to the distribution of your changes under the project's license.
+
+LGPLv3 forbids modifying source code without sharing the changes under the same license: use other people's work, and share yours!
+
+This constraint does not propagate through APIs: You can use OSRD as a library, framework or API server to interface with proprietary software. Please suggest changes if you need new interfaces.
+
+## Set things up
+
+{{% alert color="info" %}}
+Most OSRD developers use Linux (incl. [WSL](https://learn.microsoft.com/en-us/windows/wsl/)). Windows and MacOS should work too, but you may run into some issues.
+{{% /alert %}}
+
+### Get the source code
+
+- Install [`git`](https://git-scm.com/).[^package-manager]
+- Open a terminal[^git-bash] in the folder where the source code of OSRD will be located
+- Run `git clone git@github.com:osrd-project/osrd`
+
+### Launch the application
+
+Thanks to `docker`, one can easily compile, configure, and run all services after making a change. One can also start only a subset of the services.
+
+- Install `docker`. [^package-manager][^docker-desktop]
+- Follow [OSRD's README](https://github.com/osrd-project/osrd#getting-started).
+
+[^package-manager]: Under Linux, use your distribution's package manager, such as `apt-get`
```

Comment:
```suggestion
[^package-manager]: Under Linux, use your distribution's package manager, such as `apt`
```

Reply:
just a suggestion

Reply:
I would go further and write out `apt install docker`